### ISSUE TITLE
Bug with data.table

### ISSUE THREAD

**Issue by ArthurPERE** [r-lib/styler#445]
Hello,
I have a bug with the newest version of styler, when I style this selection :
![capture du 2018-11-22 10-09-18](https://user-images.githubusercontent.com/9747808/48892599-b9f4a080-ee3e-11e8-92c0-53fedacc8b68.png)

the result is this :
![capture du 2018-11-22 10-10-39](https://user-images.githubusercontent.com/9747808/48892677-e5778b00-ee3e-11e8-9f02-8e910750c1a3.png)

For data.table and all kind of the data frame, this code bug. I don't know the reason for this bug.

Thank you,
Arthur

**Comment by lorenzwalthert** [comment]
Thanks for reporting this. I believe the problem is #402, will look into later this asap. For now, you should be able to work around this by installing the old styler version with 
``` r
remotes::install_github("r-lib/styler@v1.0.2")
```


or put all code on one line. Sorry for the inconvenience.

**Comment by lorenzwalthert** [comment]
Fixed in #466. You can now also use the devel version of styler with all features introduced in 1.1.0 instead of the old release.
```r
remotes::install_github("r-lib/styler")
```

**Comment by ArthurPERE** [comment]
thank you
